view/theme/frio/templates/jot.tpl

```diff
@@ -118,6 +122,13 @@
 				</div>
 
 				<div id="jot-preview-content" class="minimize" aria-labelledby="jot-preview-lnk" role="tabpanel" aria-hidden="true"></div>
+
+				<div id="jot-preview-share" class="minimize" aria-labelledby="jot-preview-lnk" role="tabpanel" aria-hidden="true">
+					<ul id="profile-jot-submit-wrapper" class="jothidden nav nav-pills">
```

Comment:
The id "profile-jot-submit-wrapper" is already in us but id's should be unique. So you should chose another id name